### ISSUE TITLE
MathJax Version

### ISSUE THREAD

**Issue by xsrvmy** [asciidoctor/asciidoctor#2190]
Right now, MathJax is at version 2.6.0. Can we change that to the newest 2.7.1?

**Comment by mojavelinux** [comment]
Yes, as long as it can be tested and verified to not introduce any issues.
